packages/react-core/src/filters/FilterTypes.js

```diff
@@ -50,10 +50,13 @@ function stringSearch(filterValues, featureValue, params = {}) {
   const normalizedFeatureValue = normalize(featureValue, params);
   const stringRegExp = filterValues
     .map((filterValue) => {
-      let stringRegExp = escapeRegExp(normalize(filterValue, params));
+      let stringRegExp = normalize(filterValue, params);
```

Comment:
That's true. I'm clueless today 🥲. If params.useRegExp is true, it avoids the map processing because it should be passed like it is.

Reply:
Well, I found out that removing the `normalize` makes the test fail because now we are not converting both strings to lowercase before comparing so maybe we should keep that

Reply:
New changes implemented 👌